### PR TITLE
added initial report to flash scripts

### LINKED ISSUE #1891: Flash script: "initial-report"

### PR DESCRIPTION
**Pull request by bthaile**
fixes https://github.com/AugurProject/augur/issues/1891

added connect to sdk in `connect` script
added calls to sdk to get needed info to call initial report on Market contract.

to use, make sure to push time so that the market is in `designated reporting` or `open reporting` state.
>yarn flash interactive
>connect -u
>initial-report -m <marketid> -o <outcome>

### CODE REVIEW

File: packages/augur-sdk/src/Augur.ts [hunk 1]
```diff
@@ -161,7 +161,7 @@ export class Augur<TProvider extends Provider = Provider> {
   }
 
   bindTo<R, P>(f: (db: any, augur: any, params: P) => Promise<R>): (params: P) => Promise<R> {
-    return Augur.connector.bindTo(f);
+    return Augur.connector && Augur.connector.bindTo(f);
```
Comment (justinbarry) [inline packages/augur-sdk/src/Augur.ts:164]: @bthaile Why did you need to do this?

Comment (bthaile) [reply packages/augur-sdk/src/Augur.ts:164]: for some reason bindTo was called when `connector` was null. it was in a test. also made sense to not `.bindTo` if connector doesn't exist.

Comment (justinbarry) [reply packages/augur-sdk/src/Augur.ts:164]: Create needs to be called first. I wonder what we were doing where that wasn't happening.

**Review by justinbarry — APPROVED** [review]
LGTM

File: packages/augur-tools/src/flash/scripts.ts [hunk 2]
```diff
@@ -344,9 +351,12 @@ export function addScripts(flash: FlashSession) {
       if (extraStake) {
         preEmptiveStake = new BigNumber(extraStake).dividedBy(QUINTILLION).toFixed();
       }
+      if (!this.usingSdk) this.log("This script needs sdk, make sure to connect with -u flag");
```
Comment (robert-e-davidson3) [inline packages/augur-tools/src/flash/scripts.ts:354]: This doesn't leave this function so it's going to error later when trying to actually do things.
